### DISCUSSION TITLE
Looking for web browser capability

### DISCUSSION THREAD

**Discussion by pocholo007** [pulsar-edit/.github#281] [Community Package Help] [answered]
I am looking for an Pulsar package to view inside the editor HTML files from localhost. I have tried several old Atom [extensions](https://web.pulsar-edit.dev/packages), like [browser-plus](https://web.pulsar-edit.dev/packages/browser-plus) or [atom-browser](https://web.pulsar-edit.dev/packages/atom-browser), but none of them seems to work with Pulsar 1.119 x64; they can not be activated and throw errors such as _Cannot find module "theorist"_ or _Cannot find module "atom-space-pen-views"_.
Do I miss some additional settings to make these old packages work or there is any new/updated package which I should use? Thank you.

**Comment by pocholo007** [comment, marked as answer]
I finally managed to install correctly the browser-plus package and seems to work properly.

The problem was related to the portable mode configuration of the Pulsar; for some reason the command `pulsar -p install <package>` does not redirect the call to the `ppm` tool, in console the output is empty, and I had to rely on `ppm install <package>`. I checked that no other `ppm` tool was called (see https://github.com/orgs/pulsar-edit/discussions/277)

**Reply by savetheclocktower** [reply]
I'm glad this got sorted out. We are in the process of fixing `pulsar -p` so that it can act as a proper proxy for `ppm`; but it's also more than OK to use `ppm` directly.
